src/components/header.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable object-curly-newline */
+import React from 'react';
+import { Avatar, Typography, Paper, Badge } from '@mui/material';
+import NotificationsIcon from '@mui/icons-material/Notifications';
+
+const Header = () => (
+  <Paper
+    sx={{
+      display: 'flex',
+      boxShadow: '1px 1px 2px gray',
+      height: '7vh',
+      alignItems: 'center',
+      justifyContent: 'flex-end',
+      padding: '0px 20px',
+      position: 'sticky',
+      top: 0,
+    }}
+  >
+    <Paper
+      elevation={0}
+      sx={{
+        display: 'flex',
+        justifyContent: 'space-between',
+        alignItems: 'center',
+        width: '200px',
+      }}
+    >
+      <Typography variant="h6" sx={{ color: '#1A2D6D' }}>
+        Richard
+      </Typography>
```

Comment:
Do not hardcode anything, use data of logged in user

Reply:
I have worked on the comment you gave me.